src/components/Slider/Slider.spec.tsx

```diff
@@ -0,0 +1,38 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { useState } from "react";
+import Slider from "./Slider";
+import css from "./Slider.module.css";
+
+const SLIDER_ITEMS = [
+    { id: "a", name: "abc" },
+    { id: "b", name: "def" },
+    { id: "c", name: "ghi" },
+];
+
+const Component = () => {
+    const [active, setActive] = useState(SLIDER_ITEMS[0].id);
+    return <Slider items={SLIDER_ITEMS} activeItemId={active} onChange={setActive} />;
+};
+
+beforeEach(() => {
+    mount(<Component />);
+    cy.get("[data-test-id=slider]").as("slider");
+    cy.get("@slider").children().first().as("firstItem");
+    cy.get("@slider").children().last().as("lastItem");
+});
+
+describe("Slider Component", () => {
```

Comment:
We could also add an icons slider in the tests as the result may change depending on texts vs icons.